src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -878,6 +909,8 @@ internal string GetBaseUrl()
             return hostUrl + notificationUri.AbsolutePath.TrimEnd('/');
         }
 
+        internal string GetRpcBaseUrl() => InternalRpcUri.OriginalString;

```

Comment:
I'm not sure I understand the benefit of the TryGet pattern here. We will always return a value.

Reply:
I definitely see your point.

My thinking was along the line of making it more clear when to use the result of this method. For instance, the only current caller of this method wraps the call with a check that `this.config.Options.LocalRpcEndpointEnabled != false`, because that is really the only case where we need this URL. The TryGet convention makes it a bit more clear to to callers that it might not make sense to use the result from this method, even if it always returns a non-null value.

Reply:
Ah, actually yes I agree this would be cleaner. I think I missed this because I only saw a small snippet of the calling code and not the lines around it. Let me look into this.